app/controllers/about_controller.rb

```diff
@@ -1,4 +1,20 @@
 class AboutController < ApplicationController
 
+  def africanpoetrybookfund
+    @title = "APBF"
+  end
+
+  def credits
+    @title = "Credits"
+  end
+
+  def home
+    @title = "About"
+  end
+
+  def technicaldetails
+    @title = "Technical Details"
+  end
+
 
```

Comment:
@jduss4 why are these titles set here rather than the locales file?

Reply:
This is a good point, we probably should be doing them this way, but these were not the only ones like this.  It will be tricky if we do support multi-languages later because item titles are in English and being assigned to `@title` on specific event / people / work pages, etc.

I think it makes sense to create a new issue to move these titles to the locale files.